src/modules/common/helpers/response-filters.ts

```diff
@@ -38,4 +38,56 @@ export const chapterListResponseFilter = (apiResponse: IChaptersAPIResponse) =>
         return chapterListItem
     })
     return chapterList
+}
+
+export const carouselFilter = (apiResponse: ITopComicsAPIResponse, category: string) => {
+    switch (category) {
+        case "new":
+            {
+                const newMangaListState: ITrendingMangaApp[] = apiResponse.news.map((manga: INews) => {
+                    return {
+                        slug: manga.slug,
+                        title: manga.title!,
```

Comment:
Maybe do `manga.title ?? ""`  not 100% sure but try and avoid the assertion.

Reply:
It worked, Thank you!